pwnlib/windbg.py

```diff
@@ -0,0 +1,431 @@
+# -*- coding: utf-8 -*-
+"""
+During exploit development, it is frequently useful to debug the
+target binary under WinDbg.
```

Comment:
You'll have to add `docs/source/windbg.rst` and reference it from `index.rst` for these docs to show up.